src/VectorDeleteByMetadataFilter.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Upstash\Vector;
+
+use Upstash\Vector\Contracts\Arrayable;
+
+final readonly class VectorDeleteByMetadataFilter implements Arrayable
```

Comment:
small question, why do we make the filter arrayable? also, where do we utilize it's toArray method?

Reply:
https://github.com/upstash/vector-php/pull/17/files#diff-d3b56e492ebcca1f7dee384c1a90e43186b20f4b553472d91e574ff35c988729R35-R37